src/main.rs

```diff
@@ -32,11 +32,13 @@ use crate::virt::{ExecutionMode, VirtContext};
 // Defined in the linker script
 extern "C" {
     pub(crate) static _stack_start: u8;
+    pub(crate) static _bss_start: u8;
+    pub(crate) static _bss_stop: u8;
 }
 
 pub(crate) extern "C" fn main(hart_id: usize, device_tree_blob_addr: usize) -> ! {
     // For now we simply park all the harts other than the boot one
-    if hart_id != 0 {
+    if hart_id != 1 {
```

Comment:
That is going to cause issue on other (non VisionFive2) platforms. For instance on a 1 core QEMU Virt board Miralis will simply stay blocked there.
A solution to fix this issue would be to have a "BOOT_HART_ID" constant on the `Platform` trait. That way the VisionFive2 platform can put 1 there and QEMU Virt 0.